packages/console/src/cloud/pages/Main/index.tsx

```diff
@@ -18,8 +19,16 @@ function Protected() {
 
   useEffect(() => {
     const loadTenants = async () => {
-      const data = await api.get('/api/tenants').json<TenantInfo[]>();
-      setTenants(data);
+      try {
+        const data = await api.get('/api/tenants').json<TenantInfo[]>();
+        setTenants(data);
+      } catch (error: unknown) {
+        if (error instanceof HTTPError && error.response.status === 401) {
+          throw error;
+        }
+        // Note: this is intended to cause the app to crash.
```

Comment:
We need to rephrase the comment. It's not for the app "to crash", but throw a non-http error instead in order to show the error details in the ErrorBoundary.

Reply:
Removed for we have a new implementation.